### PR TITLE
[T3A3][W10-A4]Wang Xien Dong

### PR DESCRIPTION
**Pull request by xdrawks**
Ready to merge

### CODE REVIEW

File: src/seedu/addressbook/commands/CommandResult.java [hunk 1]
```diff
@@ -32,5 +32,12 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
     public Optional<List<? extends ReadOnlyPerson>> getRelevantPersons() {
         return Optional.ofNullable(relevantPersons);
     }
+    
+    /**
+     * Gets the feedback to the user.
+     */
+    public String getFeedbackToUser(){
+    	return this.feedbackToUser;
+    }
```
Comment (damithc) [inline src/seedu/addressbook/commands/CommandResult.java]: Some comments added. Please close the PR after reading comments.

File: src/seedu/addressbook/data/person/Address.java [hunk 2]
```diff
@@ -37,19 +77,51 @@ public static boolean isValidAddress(String test) {
...
+        
+        StringBuilder addressValue = new StringBuilder();
+        String divider = ", ";
+        
+        for (int i = 0; i < addressComponents.size(); i++){
+            addressValue.append(addressComponents.get(i));
+            if (i != addressComponents.size() - 1) {
+                addressValue.append(divider);
+            }
+        }
```
Comment (damithc) [inline src/seedu/addressbook/data/person/Address.java]: `StringJoiner` may be useful here.

File: src/seedu/addressbook/data/person/Address.java [hunk 3]
```diff
@@ -37,19 +77,51 @@ public static boolean isValidAddress(String test) {
...
 
     @Override
     public boolean equals(Object other) {
         return other == this // short circuit if same object
                 || (other instanceof Address // instanceof handles nulls
-                && this.value.equals(((Address) other).value)); // state check
+                && Objects.equals(this.block, ((Address) other).block)
+                    && Objects.equals(this.street, ((Address) other).street)
+                            && Objects.equals(this.street, ((Address) other).street)
+                                    && Objects.equals(this.postalCode, ((Address) other).postalCode)); // state every single address component.
```
Comment (damithc) [inline src/seedu/addressbook/data/person/Address.java]: Any reason why this is indented as a staircase?

**Comment by damithc** [comment]
Good effort @xdrawks Some comments added. Please close the PR after reading comments.

File: src/seedu/addressbook/data/person/Address.java (hunk 2 above)
Comment (xdrawks) [inline src/seedu/addressbook/data/person/Address.java]: Oh, that's great! Thanks for suggesting.

File: src/seedu/addressbook/data/person/Address.java (hunk 3 above)
Comment (xdrawks) [inline src/seedu/addressbook/data/person/Address.java]: The staircase was auto-generated by Eclipse and I thought it was meant to be as such.

**Pull request closed without merging**